server/app/middleware/question.js

```diff
@@ -0,0 +1,87 @@
+import {
+  getAll,
+  getOne,
+  postQuestion,
+  postAnswer,
+  deleteQuestion,
+  favAnswer,
+} from './../controller/question';
+/** Get all question
+ * @param {obj}
+ * @return {obj}
+ * @public
+*/
+const allquestion = (req, res) => {
+  getAll().then((questions) => {
+    res.send({
+      results: questions,
+    });
+  }).catch(() => res.status(400).send({ error: 'An Error Occured' }));
```

Comment:
You should use the structure described earlier for all your responses to the client.